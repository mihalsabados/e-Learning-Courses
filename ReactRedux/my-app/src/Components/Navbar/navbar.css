.navbar{
    height: 80px;
    width: 100%;
    background-color: #0164b2;
    display: flex;
    justify-content: right;
}

.logo-div{
    margin-right: 50px;
}

.logo-div img{
    height: 80px;
}