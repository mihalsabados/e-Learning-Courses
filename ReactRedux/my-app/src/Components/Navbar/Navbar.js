import './navbar.css';

export default function Navbar() {
    return (
        <div className="navbar"> 
            <div className="logo-div">
                <img src="resources/logo.png" alt='logo'/>
            </div>
        </div>
    );
  }