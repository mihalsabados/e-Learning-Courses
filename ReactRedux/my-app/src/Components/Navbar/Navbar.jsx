import React from "react";
import "./Navbar.css";

export default function Navbar() {
	return (
		<div className="navbar">
			<div className="logo-div">
				<img src="resources/logo.png" alt="logo" />
			</div>
		</div>
	);
}
