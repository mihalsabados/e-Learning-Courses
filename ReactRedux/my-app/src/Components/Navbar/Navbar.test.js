/* eslint-disable no-undef */
import React from "react";
import { render, screen } from "@testing-library/react";
import Navbar from "./Navbar";
import renderer from "react-test-renderer";

test("loads and displays logo", async () => {
	render(<Navbar />);

	// Assert that the logo image is rendered with the correct alt text
	expect(screen.getByAltText("logo")).toBeInTheDocument();
});

test("Navbar component snapshot", () => {
	const component = renderer.create(<Navbar />);
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
