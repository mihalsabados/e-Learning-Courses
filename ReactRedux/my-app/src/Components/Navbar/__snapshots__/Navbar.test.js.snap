// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb component snapshot 1`] = `
<div
  className="navbar"
>
  <div
    className="logo-div"
  >
    <img
      alt="logo"
      src="resources/logo.png"
    />
  </div>
</div>
`;

exports[`Navbar component snapshot 1`] = `
<div
  className="navbar"
>
  <div
    className="logo-div"
  >
    <img
      alt="logo"
      src="resources/logo.png"
    />
  </div>
</div>
`;
